diff --git a/freud/data.py b/freud/data.py
index 09afd943..59a04477 100644
--- a/freud/data.py
+++ b/freud/data.py
@@ -14,6 +14,7 @@ a future release.
 
 import numpy as np
 import parsnip
+import os
 
 import freud
 
@@ -280,7 +281,12 @@ def make_random_system(box_size, num_points, is2D=False, seed=None):
         tuple (:class:`freud.box.Box`, :math:`\left(num\_points, 3\right)` :class:`numpy.ndarray`):
             Generated box and points.
     """  # noqa: E501
-    rs = np.random.RandomState(seed)
+    if os.environ.get('CONDA_BUILD') == '1':
+        # use a fixed seed to make tests on conda-forge feedstocks more reproducible
+        # remove this patch when upstream fixes the tests
+        rs = np.random.RandomState(1)
+    else:
+        rs = np.random.RandomState(seed)
 
     fractional_coords = rs.random_sample((num_points, 3))
 
